declarations_site/spotter/views.py

```diff
@@ -61,17 +57,41 @@ def save_search(request):
         return redirect('search_list')
 
     task = SearchTask(user=request.user, query=query, deepsearch=deepsearch)
+    task.query_params = query_params
     task.save()
 
-    first_run(task)
-    if not send_newtask_notify(task):
+    if not first_run(task):
+        messages.warning(request, 'Не вдалось створити завдання, спробуйте спростити '+
```

Comment:
Well, more or less:
If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies). Use your own judgment; however, never use more than one space, and always have the same amount of whitespace on both sides of a binary operator.